Simplenote/Classes/FileManager+Simplenote.swift

```diff
@@ -14,19 +14,18 @@ extension FileManager {
 
         return url
     }
-
-    /// Writes a given Note to the Documents folder
+    
+    /// User's Temporary Directory
     ///
-    class func writeToDocuments(note: Note) -> URL? {
-        guard let payload = note.content else {
-            return nil
-        }
-
-        let filename = String(format: "%@.txt", note.simperiumKey)
-        let targetURL = FileManager.documentsURL.appendingPathComponent(filename)
-
+    class var temporaryDirectoryURL: URL {
+        return FileManager.default.temporaryDirectory
+    }
```

Comment:
No need to wrap access to the `temporaryDirectory` (since it's accessible from the SDK singleton)